src/views/scans/viewScansList.tsx

```diff
@@ -193,18 +150,20 @@ const ScansListView: React.FunctionComponent = () => {
     </Toolbar>
   );
 
-  const renderConnection = (scan: ScanType): React.ReactNode => {
+  const renderConnection = (scan: Scan): React.ReactNode => {
```

Comment:
@cdcabrera I also spotted another little bug related to this. The scan name was not being rendered when opening the modal for last scanned. That made me change a few things in the showScansModal and consequently now the tests related to that have extra reasons to fail. I didn't touch it as we will be rewriting them, right ? Just a heads up whenever u see the changes.